### PR TITLE
Extended support by implementing TDIGEST.QUANTILE command

### LINKED ISSUE #1068: Add support for TDIGEST.QUANTILE bloom command
The associated API is documented at https://redis.io/commands/TDIGEST.QUANTILE

### PR DESCRIPTION
**Pull request by shacharPash**
Closes #1068 
@vladvildanov

### CODE REVIEW

File: tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        return TDIGESTQUANTILE::class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedId(): string
+    {
+        return 'TDIGESTQUANTILE';
+    }
```
Comment (vladvildanov) [inline tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php:34]: Missing `testFilterArguments()`

File: tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        $this->assertSame(1, $this->getCommand()->parseResponse(1));
+    }
+
+    /**
+     * @group connected
+     * @return void
+     * @requiresRedisBfVersion >= 2.4.0
+     */
+    public function testquantile(): void
```
Comment (vladvildanov) [inline tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php:49]: 1. Test name should be camelCased
2. Test name should be more specific to represent what we're actually testing, something like `testReturnsValuesBelowGivenQuantile`

File: tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    {
+        $redis = $this->getClient();
+
+        $redis->tdigestcreate('key', 1000);
+
+        $addResponse = $redis->tdigestadd('key', 1, 2, 2, 3, 3, 3, 4, 4, 4, 4, 5, 5, 5, 5, 5);
+        $quantileResponse = $redis->tdigestquantile('key', 0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0);
+
+        $this->assertEquals('OK', $addResponse);
+        $this->assertSame(['1', '2', '3', '3', '4', '4', '4', '5', '5', '5', '5'], $quantileResponse);
```
Comment (vladvildanov) [inline tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php:59]: Add one more assertion to test empty TDigest sketch:
`$redis->tdigestcreate('empty_key');`
`$this->assertSame(['nan', 'nan'], $redis->tdigestquantile('empty_key', 0.0, 0.1));`

**Review by vladvildanov — CHANGES_REQUESTED** [review]
